### PR TITLE
Add metrics for account balance (billing).

### PR DESCRIPTION
**Pull request by weikinhuang**
Add gauges to monitor the current billing activity.
See [balance api](https://developers.digitalocean.com/documentation/v2/#balance)

### CODE REVIEW

**Comment by circa10a** [comment]
@weikinhuang can you also update the readme with the added metrics?

**Comment by weikinhuang** [comment]
Sure thing!

**Comment by weikinhuang** [comment]
updated

File: README.md [hunk 1]
```diff
@@ -67,14 +69,16 @@ Read-only tokens are sufficient.
...
+| digitalocean_month_to_date_usage            | gauge   | 1            | Amount used in the current billing period as of the `digitalocean_balance_generated_at` time
 | digitalocean_snapshot_min_disk_size_bytes   | gauge   | 2            | Minimum disk size for a droplet/volume to run this snapshot on in bytes
 | digitalocean_snapshot_size_bytes            | gauge   | 2            | Snapshot's size in bytes
 | digitalocean_start_time                     | gauge   | 1            | Unix timestamp of the start time
 | digitalocean_volume_size_bytes              | gauge   | 11           | Volume's size in bytes
 
 ### Alerts & Recording Rules
 
-As example alerts and recording rules I have copied my `.rules` file to this repository.  
+As example alerts and recording rules I have copied my `.rules` file to this repository.
```
Comment (metalmatze) [inline README.md:81]: This has been an intentional new line break with the 2 spaces at the end.

Comment (weikinhuang) [reply README.md:81]: i'll fix those, my editor auto trims whitespace

**Comment by metalmatze** [comment]
Very nice!
I was about to comment that the metrics should end with `_dollars`, however the metrics I created don't have that suffix either.
Let's merge it as is then.

**Review by metalmatze — APPROVED** [review]

**Comment by weikinhuang** [comment]
Thanks!

**Pull request merged**